apps/DemoApp/DemoApp/PlayerViewController.swift

```diff
@@ -222,6 +232,38 @@ class PlayerViewController: UIViewController {
         MUXSDKStats.videoChangeForPlayer(name: self.playerName, customerData: customerData)
     }
     
+    @objc func MUXProgramChange() {
+        let playerData = MUXSDKCustomerPlayerData(environmentKey: "shqcbkagevf0r4jh9joir48kp")
```

Comment:
and this?
i prefer to just have a single var to store the env key in and reference it :P

Reply:
yeah much better, I've updated.